types/webpack/package.json

```diff
@@ -0,0 +1,5 @@
+{
+    "dependencies": {
```

Comment:
Set this to private.

Reply:
_sigh_ That's another thing that causes the local linter to fail but apparently is required in CI. Either I'm using the local linter wrong (just `npm run lint webpack`, right?), or the README needs to be updated.

Reply:
Yes, thats the correct command.